src/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs

```diff
@@ -173,6 +161,69 @@ internal static bool IsEnabled()
 
         #region private
 
+        private const string SuppressActivityPropagationEnvironmentVariableSettingName = "DOTNET_SYSTEM_NET_HTTP_SUPPRESSACTIVITYPROPAGATION";
+        private const string SuppressActivityPropagationAppCtxSettingName = "System.Net.Http.SuppressActivityPropagation";
```

Comment:
I'd recommend we use affirmative terms rather than negated terms to make it easier to reason about. So for example:
SuppressActivityPropagation -> ActivityPropagation or EnableActivityPropagation.

For environment variables it always possible to make the default setting when unspecified be 'true' or '1' so you can still have same defaults you had now.